pkg/controller/kibana/controller.go

```diff
@@ -65,13 +66,18 @@ var (
 func (r *ReconcileKibana) Reconcile(request reconcile.Request) (reconcile.Result, error) {
 	es, err := k8shandler.GetElasticsearchCR(r.client, request.Namespace)
 	if err != nil {
-		return reconcileResult, fmt.Errorf("skipping kibana reconciliation in %q: %s", request.Namespace, err)
+		logrus.Infof("skipping kibana reconciliation in %q: %s", request.Namespace, err)
+		return reconcileResult, nil
 	}
 
 	esClient := elasticsearch.NewClient(es.Name, es.Namespace, r.client)
 	if err := kibana.Reconcile(request, r.client, esClient); err != nil {
-		return reconcileResult, err
+		if errors.IsNotFound(err) {
+			return reconcile.Result{Requeue: true}, nil
```

Comment:
Why not reconciling after a specific period here?

Reply:
i guess this reconciliation is immediate, earlier than 30 secs wait.

Reply:
Ok if this the intention, I am fine.